packages/material-ui/src/test-utils/describeConformance.js

```diff
@@ -176,6 +177,13 @@ const fullSuite = {
 export default function describeConformance(minimalElement, getOptions) {
   const { only = Object.keys(fullSuite), skip = [] } = getOptions();
   describe('Material-UI component API', () => {
+    after(() => {
+      const { mount } = getOptions();
+      if (mount.attachTo) {
```

Comment:
When would this be falsy?

Reply:
This is undefined in the Table tests e.g.
https://github.com/mui-org/material-ui/blob/7fb9b403553bb9d74f1ef00c07f53e61341cc71e/packages/material-ui/src/TableCell/TableCell.test.js#L10

Reply:
Shouldn't the test be fixed then? The nodes should be inserted into an actual document not act as zombies.

Reply:
The node is inserted into an actual document. I'm not aware of any zombie node. 

In all the occurrence of the table component, it shouldn't be a problem. The cleanup in the test description will take care of removing the dom element.

This logic was added in order to work around an issue with the getByRole logic. Here is the problem in more detail. The describeConformance mounts a slider in the page. The testing-library also mounts a slider in the page. We end up with two sliders in the page. `getByRole` find two role elements instead of one (the current element we want to test). It throws because there are two elements when we only expect one.

Reply:
I have rebased, the problem seems to be solved. I'm removing these lines.